SamplesNew/login.php

```diff
@@ -51,6 +51,18 @@ function handleTan(\Fhp\BaseAction $action)
         echo 'Please use this device: ' . $tanRequest->getTanMediumName() . "\n";
     }
 
+    // Challenge Image for PhotoTan/ChipTan
+    if($tanRequest->getChallengeHhdUc())
```

Comment:
Missing space: `if (`
Does php-cs not complain?

Reply:
Also we don't normally put the `{` of `if`s on a new line here.

Reply:
It's php-cs complain. I do not see the benefit of having this space. Please push a change on php-cs declaration to prevent this issue again. It's not fun to update a pull request because of thinks like these.

Reply:
I agree that updating because of this isn't fun. But there are other issues like this that are less tolerable (e.g. wrong parameter name in a `@param` in phpDoc) and also would require updating. I assume you don't regularly use an auto-formatter, at least when working on this project? Maybe we can have a pre-commit hook that runs php-cs? Also, I wonder why Travis didn't complain.

Reply:
Ok, this was unlucky: My local execution of php-cs-fixer was only partly executed and i didn't realised that the single warning "Files that were not fixed due to errors reported during linting before fixing" is a fatal error. This happened because the default PHP5 binary of my Mac is not helping when linting the PHP7 formatted files. As php-cs-fixer has fixed an issue in a file, that was not touched by myself and the Travis test results were valid, i assumed to be on the right way. Sorry for that. I analysed why Travis did not recognise that and pushed #230.